src/components/experimental/Calendar/Calendar.styled.ts

```diff
@@ -1,131 +1,251 @@
+    --rdp-weekday-text-align: center;
 
-export const Button = styled(BaseButton)`
-    appearance: none;
-    background: none;
-    border: none;
-    display: flex;
-    cursor: pointer;
-    margin: 0;
-    padding: 0;
     color: ${getSemanticValue('on-surface')};
-    outline: 0;
 
-    &[data-focused] {
-        outline: ${getSemanticValue('interactive')} solid 0.125rem;
-        border-radius: ${get('radii.2')};
+    .rdp {
+        width: fit-content;
     }
 
-    &[data-disabled] {
-        opacity: 0;
+    /* Layout for multiple months */
+    .rdp-months {
+        display: flex;
+        flex-direction: row;
+        gap: var(--rdp-months-gap);
+        position: relative;
     }
-`;
 
-export const Heading = styled(BaseHeading)`
-    margin: 0;
-    color: ${getSemanticValue('on-surface')};
-    font-size: var(--wave-exp-typescale-title-2-size);
-    font-weight: var(--wave-exp-typescale-title-2-weight);
-    line-height: var(--wave-exp-typescale-title-2-line-height);
-`;
+    .rdp-month {
+        display: flex;
+        flex-direction: column;
+        gap: ${get('space.3')};
+    }
 
-export const CalendarGrid = styled(BaseCalendarGrid)`
-    border-collapse: separate;
-    border-spacing: 0 0.125rem;
+    /* Navigation */
+    .rdp-nav {
+        position: absolute;
+        inset-inline: 0;
+        top: 0;
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        gap: ${get('space.1')};
+        pointer-events: none; /* allow buttons only */
+        height: var(--rdp-nav-height);
+    }
 
-    td {
+    .rdp-button_previous,
+    .rdp-button_next {
+        appearance: none;
+        background: none;
+        border: 0;
+        display: inline-flex;
+        align-items: center;
+        justify-content: center;
+        width: var(--rdp-nav_button-width);
+        height: var(--rdp-nav_button-height);
         padding: 0;
+        color: ${getSemanticValue('on-surface')};
+        border-radius: ${get('radii.2')};
+        pointer-events: auto;
+        cursor: pointer;
     }
 
-    th {
-        padding: 0 0 ${get('space.1')};
+    .rdp-button_previous:focus-visible,
+    .rdp-button_next:focus-visible {
+        outline: ${getSemanticValue('interactive')} solid 0.125rem;
     }
-`;
 
-export const WeekDay = styled(CalendarHeaderCell)`
-    color: ${getSemanticValue('on-surface')};
-    font-size: var(--wave-exp-typescale-label-2-size);
-    font-weight: var(--wave-exp-typescale-label-2-weight);
-    line-height: var(--wave-exp-typescale-label-2-line-height);
-`;
+    .rdp-button_previous:disabled,
+    .rdp-button_next:disabled {
+        opacity: var(--rdp-nav_button-disabled-opacity);
+    }
 
-export const MonthGrid = styled.div`
-    display: flex;
-    gap: 1.5rem;
+    .rdp-caption_label {
+        margin: 0;
+        color: ${getSemanticValue('on-surface')};
+        font-size: var(--wave-exp-typescale-title-2-size);
```

Comment:
do we have/should we add semantic variables for this? if we don't have it already, it might not be in the scope of this MR, but maybe we can create a ticket? or does it not really make sense to do so?